superset/reports/commands/execute.py

```diff
@@ -342,23 +343,34 @@ def _get_notification_content(self) -> NotificationContent:
         ):
             embedded_data = self._get_embedded_data()
 
+        notification_source = None
+        notification_source_id = None
```

Comment:
nit, but can you declare these two at the beginning of the method with the others? It read at first as if you were reassigning the values rather than declaring and I had to check the top of the method to be sure which one it was.